lib/views/general/_responsive_footer.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="footer" id="footer">
```

Comment:
I think the template and the general layout could be merged in to [mysociety/alaveteli](https://github.com/mysociety/alaveteli). I'm keen that we keep the divergence between alaveteli and alavetelitheme to a minimum.